drms/tests/test_to_datetime.py

```diff
@@ -28,30 +27,15 @@ def test_tai_string(time_string, expected):
     ('2010-05-01 01:23:45Z', pd.Timestamp('2010-05-01 01:23:45')),
     ('2013-12-21 23:32Z', pd.Timestamp('2013-12-21 23:32:00')),
     ('2013-12-21 23:32:34Z', pd.Timestamp('2013-12-21 23:32:34')),
-    ])
+])
 def test_z_string(time_string, expected):
     assert drms.to_datetime(time_string) == expected
 
 
-@pytest.mark.skip('pandas does not support leap seconds')
-@pytest.mark.parametrize('time_string, expected', [
-    ('2012-06-30T23:59:60Z', '2012-06-30 23:59:60'),
-    ('2015-06-30T23:59:60Z', '2015-06-30 23:59:60'),
-    ('2016-12-31T23:59:60Z', '2016-12-31 23:59:60'),
-    ])
-def test_z_leap_string(time_string, expected):
-    assert drms.to_datetime(time_string) == expected
-
```

Comment:
This was skipped and pandas still doesn't support this so I removed it.

Reply:
Pandas and Python's `datetime` do not support leap seconds (see <https://bugs.python.org/issue23574> for example).

But this is still a regression, that can be triggered when using `drms.to_datetime()`, so it should not be removed from tests.

Reply:
I will add the tests back.

Reply:
Done.